pkg/virt-handler/options_test.go

```diff
@@ -0,0 +1,175 @@
+package virthandler
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gstruct"
+
+	"libvirt.org/go/libvirtxml"
+)
+
+const memorySize = 16256896
```

Comment:
Can you explain why this specific number? I suggest either adding a comment that clarifies that, or even better, assign it to the actual calculation that's taking place (i.e. `const memorySize = 2 * 1024 * 1024`)

Reply:
Arbitrary, changed.